rest/utilities_testing.go

```diff
@@ -242,6 +242,7 @@ func (rt *RestTester) Close() {
 	}
 	if rt.RestTesterServerContext != nil {
 		rt.RestTesterServerContext.Close()
+		rt.RestTesterServerContext = nil
```

Comment:
As discussed, this seems prone to races if there is scenario where multiple closes could be called.